INN/umbrella-sfpublicpress
# Make sidebar posts flow vertically, add clear:both; to bottom widget

## Changes

This pull request makes the following changes:

- Fixes podcasts in the sidebar so now they flow vertically instead of horizontally
- Adds `clear: both;` to article bottom widget area so it's not consumed by the sidebar widget area

Before:
![screencapture-sfpp-test-2020-03-02-17_26_33](https://user-images.githubusercontent.com/18353636/75723867-25b4f280-5cab-11ea-9329-69b8bb0491b2.png)

After:
![screencapture-sfpp-test-2020-03-02-17_26_18](https://user-images.githubusercontent.com/18353636/75723850-2188d500-5cab-11ea-9e90-0abd16024765.png)

## Why



For #59 

## Testing/Questions

Features that this PR affects:

- Article page widget areas and podcasts widget


Questions that need to be answered before merging:


Steps to test this PR:

1. Create/use a post using the two column template and add some widgets to the sidebar and article bottom widget area
2. Make sure on the frontend that all widgets stay in their respective widget area
